pandas/tests/series/methods/test_update.py

```diff
@@ -46,7 +46,7 @@ def test_update(self, using_copy_on_write):
         "other, dtype, expected, warn",
         [
             # other is int
-            ([61, 63], "int32", Series([10, 61, 12], dtype="int32"), None),
+            ([61, 63], "int32", Series([10, 61, 12]), FutureWarning),
```

Comment:
this would currently upcast, but at least there's a futurewarning noting that it won't in the future